### DISCUSSION TITLE
Is Nativewind compatible with AWS Amplify deployment?

### DISCUSSION THREAD

**Discussion by madmonkey08** [nativewind/nativewind#908] [Q&A]
Hi everyone, I'm working with Nativewind using the Solito starter and this is an amazing framework! I've not problem working with it in local and doing the deployment with Vercel, when I see the result with those two (Vercel and local), the styles are perfect, but when I deploy the NextJS app in Amplify, I don't know why, Amplify change the spacing of the external View, adding or removing in some cases the space that I code (padding, margin, etc). My question is, Has anyone else had problems with Nativewind deploying with amplify? Is Nativewind compatible with Amplify? Is kind of weird that locally and in Vercel's deployment is perfect and only amplify changes my styles. Thank you so much!

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
